src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -920,6 +921,19 @@ public void GetProcesses_EmptyMachineName_ThrowsArgumentException()
             AssertExtensions.Throws<ArgumentException>(null, () => Process.GetProcesses(""));
         }
 
+        [Fact]
+        public void GetProcesses_InvalidMachineName_ThrowsInvalidOperationException()
+        {
+            Assert.Throws<InvalidOperationException>(() => Process.GetProcesses(Guid.NewGuid().ToString()));
+        }
+
+        [Fact]
+        public void GetProcesses_RemoteMachinePath_ReturnsExpected()
```

Comment:
@danmosemsft this test is failling with:

> System.DirectoryServices.ActiveDirectory.ActiveDirectoryObjectNotFoundException : The local computer is not joined to a domain or the domain cannot be contacted.

As well as (On Windows.10.Amd64.ClientRS2.Open-Debug-x64)

> System.PlatformNotSupportedException : System.DirectoryServices is not supported on this platform.

Have I written something wrong?

Reply:
Ah - I would catch those two exceptions with a note as to why, and then at least it will give coverage for devs who are joined to the domain.

There may be a way to do this when not joined to the domain but I don't know what it is.